src/tests/system/tests/test_sss_override.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.AD)
+def test_sss_overrides__overriding_username_and_posix_attributes(client: Client, provider: GenericProvider):
+    """
+    :title: Locally overriding the name and POSIX attributes of a user
+    :setup:
+        1. Create POSIX user "user1" with standard POSIX attributes defined
+        2. Start SSSD
+    :steps:
+        1. Create local override for "user1" and name it "o-user1"
+        2. Authenticate as "user1", the short and fully qualified name
+        3. Authenticate as "o-user1", the short and fully qualified name
+        4. Query the user "user1" and then override the POSIX attributes
+        5. Query the username "user1", and local override name, "o-user1"
+    :expectedresults:
+        1. Local override is created for "user1"
+        2. Authentication successful for both short and fully qualified name
+        3. Authentication successful for both short and fully qualified name
+        4. POSIX attributes for local override has been changed
+        5. The name and overriden name is found and POSIX attributes are updated
+    :customerscenario: False
+    """
+    provider.user("user1").add(
+        uid=999011, gid=999011, home="/home/user1", gecos="user", shell="/bin/bash", password="Secret123"
+    )
+
+    client.sssd.domain["ldap_id_mapping"] = "False"
+    client.sssd.config_apply()
+    client.sssd.start()
+    client.sss_override.user("user1").add(name="o-user1")
+    client.sssd.restart()
+
+    result_ssh1 = client.auth.ssh.password("user1", "Secret123")
+    result_ssh2 = client.auth.ssh.password("o-user1", "Secret123")
+    result_fqn_ssh1 = client.auth.ssh.password(f"user1@{client.sssd.default_domain}", "Secret123")
+    result_fqn_ssh2 = client.auth.ssh.password(f"o-user1@{client.sssd.default_domain}", "Secret123")
+
+    result_getent1 = client.tools.getent.passwd("o-user1")
+    client.sss_override.user("user1").add(name="o-user1", uid=999999, gid=888888, home="/home/o-user1")
```

Comment:
You can use `user = client.sss_override.user(...).add(...)` and then `user.delete()`